### PR TITLE
fix: set vertical layout in BulkEditor

### PR DESCRIPTION
**Pull request by jansule**
## Description

This wraps the form items in `<BulkEditor>` into a `<Form>` and sets its layout to `'vertical'` so that the form layout will be consistent with all the other forms. It also adjusts the layout of the symbol selection.

image

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

### CODE REVIEW

**Review by KaiVolland — COMMENTED** [review]
I would avoid using form at all here as the `<form>` is not a layout element in the first place but influences the handling a lot.

Maybe you can have a look at the other editors to achieve the same result without using a form.

**Comment by jansule** [comment]
> I would avoid using form at all here as the `<form>` is not a layout element in the first place but influences the handling a lot.
> 
> Maybe you can have a look at the other editors to achieve the same result without using a form.

The other editors are all wrapped in forms as well. But I agree, the form was mainly added for using the layout property, which we also can achieve via css only.

**Comment by jansule** [comment]
Removed the forms. @KaiVolland could you take another look?

File: src/Component/RuleGenerator/RuleGenerator.tsx [hunk 1]
```diff
@@ -171,146 +172,157 @@ export const RuleGenerator: React.FC<RuleGeneratorProps> = (props) => {
...
+              ? 'error'
               : undefined
+        }
+        help={classificationMethod === 'kmeans'
+          ? locale.numberOfRulesViaKmeans
+          : numberOfRules < minNrClasses
+            // eslint-disable-next-line max-len
+            ? `${locale.colorRampMinClassesWarningPre} ${minNrClasses} ${locale.colorRampMinClassesWarningPost}`
+            : undefined
+        }
```
Comment (KaiVolland) [inline src/Component/RuleGenerator/RuleGenerator.tsx:234]: I know this is not part of thos PR but would be nice if we would have this human readable.

File: src/Util/FormItemUtil.spec.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { getFormItemConfig } from './FormItemUtil';
+
+describe('FormItemUtil', () => {
```
Comment (KaiVolland) [inline src/Util/FormItemUtil.spec.ts:31]: This test is not necessary.

**Review by KaiVolland — APPROVED** [review]

**Pull request merged**